std/testing/runner.ts

```diff
@@ -191,44 +191,20 @@ export async function runTestModules({
     return;
   }
 
-  // Create temporary test file which contains
-  // all matched modules as import statements.
-  const testFile = renderTestFile(testModules);
-  // Select where temporary test file will be stored.
-  // If `DENO_DIR` is set it means that user intentionally wants to store
-  // modules there - so it's a sane default to store there.
-  // Fallback is current directory which again seems like a sane default,
-  // user is probably working on project in this directory or even
-  // cd'ed into current directory to quickly run test from this directory.
-  const root = Deno.env("DENO_DIR") || Deno.cwd();
-  const testFilePath = join(root, ".deno.test.ts");
-  const data = new TextEncoder().encode(testFile);
-  await Deno.writeFile(testFilePath, data);
+  // Create temporary module which statically imports all matched modules.
+  const testFilePath = join(cwd(), ".deno.test.js");
```

Comment:
Hmmm no, this is bad, using `.js` prevents the optimization that all `.ts` imports are compiled in one go. Using `.js` Deno they will imported and compile each .ts` import one-by-one.